tests/e2e/specs/onboarding/step-3-complete-campaign.test.js

```diff
@@ -495,6 +495,64 @@ test.describe( 'Complete your campaign', () => {
 		} );
 	} );
 
+	test.describe( 'Enhanced conversion prompt should appear as last step in setup success modal', () => {
```

Comment:
@ankitrox Let's specify that the EC prompt should be the last step in the modal if ads setup is complete. also, let's add another test case when ads setup is not complete.